application/modules/front/views/search/index.php

```diff
@@ -79,23 +79,20 @@ function truncateHadithText($hadith)
             if ((bool)$arabicEntry) $permalink = $arabicEntry->permalink;
             else $permalink = $englishEntry->permalink;
 
-            $truncation = false;
+            [$arabicText, $arabicTruncation] = truncateHadithText($data['ar']);
```

Comment:
Changes to this file: Show hadith text even if preview is not available (match on some either criteria, such as collection and hadithNumber)